crates/relayer/src/config/types.rs

```diff
@@ -175,6 +179,164 @@ pub mod max_tx_size {
     }
 }
 
+pub mod ics20_max_receiver_size {
+    flex_error::define_error! {
+        Error {
+            TooBig
+                { value: usize }
+                |e| {
+                    format_args!("`ics20_max_receiver_size` must be less than or equal to {}, found {}",
+                    Ics20MaxReceiverSize::MAX_BOUND, e.value)
+                },
+        }
+    }
+
+    #[derive(Debug, Clone, Copy, PartialEq, Eq, PartialOrd, Ord)]
+    pub struct Ics20MaxReceiverSize(usize);
+
+    impl Ics20MaxReceiverSize {
+        /// Use limit of 2048 bytes used by ibc-go v8 as default
+        /// See: https://github.com/cosmos/ibc-go/blob/04e14a7079e18323c64d4e00e12cf77708d24701/modules/apps/transfer/types/msgs.go#L16
+        const DEFAULT: usize = 2048;
+        const MAX_BOUND: usize = 1048576; // 1 MBytes
+
+        pub fn new(value: usize) -> Result<Self, Error> {
+            if value > Self::MAX_BOUND {
+                return Err(Error::too_big(value));
+            }
+
+            Ok(Self(value))
+        }
+
+        pub fn max() -> Self {
+            Self(Self::MAX_BOUND)
+        }
+
+        pub fn to_usize(self) -> usize {
+            self.0
+        }
+    }
+
+    impl Default for Ics20MaxReceiverSize {
+        fn default() -> Self {
+            Self(Self::DEFAULT)
+        }
+    }
+
+    use serde::de::Unexpected;
+    use serde::{de::Error as _, Deserialize, Deserializer, Serialize, Serializer};
+
+    impl<'de> Deserialize<'de> for Ics20MaxReceiverSize {
+        fn deserialize<D>(deserializer: D) -> Result<Self, D::Error>
+        where
+            D: Deserializer<'de>,
+        {
+            let value = usize::deserialize(deserializer)?;
```

Comment:
Let's use `byte_unit::Byte` like we do for `max_grpc_decoding_size`.

Reply:
Fixed in 43a79a35c048b1eeef4d50114b2f2e060154fb4e